login_page.php

```diff
@@ -241,7 +247,10 @@ function debug_setting_message ( $p_type, $p_setting, $p_value ) {
 			</ul>
 			<div class="field-container">
 				<label for="username"><span><?php echo lang_get( 'username' ) ?></span></label>
-				<span class="input"><input id="username" type="text" name="username" size="32" maxlength="<?php echo DB_FIELD_SIZE_USERNAME;?>" value="<?php echo string_attribute( $f_username ); ?>" class="<?php echo $t_username_field_autofocus ?>" /></span>
+				<span class="input">
+					<input id="username" type="text" name="username" size="32" maxlength="<?php
+						echo DB_FIELD_SIZE_USERNAME;?>" value="<?php echo string_attribute( $f_username ); ?>" class="<?php
+						echo $t_username_field_autofocus ?>" <?php echo $t_username_placeholder; ?> /></span>
```

Comment:
Now I change the label rather than add a place holder.  Though not because of IE9 :) -- just checked evernote.com and they have it in the label.  I remember seeing it else where in the place holder.